source/manual/mongodb.html.md

```diff
@@ -5,29 +5,13 @@ layout: manual_layout
 parent: "/manual.html"
 section: Backups
 type: learn
-last_reviewed_on: 2020-03-02
+last_reviewed_on: 2020-10-13
 review_in: 6 months
 ---
 
-There are two ways of taking MongoDB backups.
-
-## automongodbbackup
-
-This is how MongoDB backups have traditionally been taken on the GOV.UK Infrastructure.
-
-A third-party script, [automongodbbackup](https://github.com/micahwedemeyer/automongobackup), takes a nightly `mongodump` and stores it on one of the MongoDB machines' dedicated mount points (likely [the first machine in the replicaset as defined in the Puppet manifest](https://github.com/alphagov/govuk-puppet/blob/master/modules/mongodb/manifests/backup.pp#L40-L44)).
-
-The on-site backup machine (`backup-1.management`) pulls the latest backup and stores it on disk. [Duplicity](http://duplicity.nongnu.org/) runs nightly and sends encrypted backups to an AWS S3 bucket.
-
-### Restoring
-
-- Fetch a backup from either the dedicated mount point, the on-site machine, or the S3 bucket [using Duplicity](restore-from-offsite-backups.html) (you may need a password kept in the encrypted [hieradata](https://github.com/alphagov/govuk-secrets)).
-- Unzip the file. This will produce a directory of data.
-- Run the command: `mongo restore --drop <directory>`
-
 ## mongodumps to S3
```

Comment:
Is this bit dead now as well? Is there anything left on the old mongodump arrangements that wasn't moved to govuk-env-sync (as described below)?

Reply:
I was about to reply 'no it's all gone' and then I remembered performance platform uses mongodb *sob*

Reply:
oh no 😭 no worries, thanks for updating all these!